src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -1004,6 +1004,18 @@ private AdsDataProvider() {
   }
 
   @DataProvider
+  public static Object[][] adsDetection() {
+    return new Object[][]{
+        {
+            "project43",
+        },
+        {
+            "arecovery"
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "arecovery" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1192)